purbee_frontend/src/components/homepage/sidebar.js

```diff
@@ -0,0 +1,74 @@
+import Drawer from '@mui/material/Drawer';
+import List from '@mui/material/List';
+import Divider from '@mui/material/Divider';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import HomeIcon from '@mui/icons-material/Home';
+import PersonIcon from '@mui/icons-material/Person';
+import { makeStyles } from '@mui/styles';
+import { Link } from 'react-router-dom';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import { purple } from '@mui/material/colors';
+
+const useStyles = makeStyles(() => ({
+    root: {
+        overflowX: "hidden",
+        overflowY: "hidden"
+    },
+    drawerHead: {
+        color: "#fff",
+        backgroundColor: purple[900],
+        height: "3.6em",
+    }
+}))
+
+export const SideBar = (props) => {
+
+    const classes = useStyles();
+
+    return (
+        <Drawer
+            open={props.open}
+            anchor="right"
+            onClose={() => { props.setOpen(false) }}
+            className={classes.root}
+        >
+            <ListItem className={classes.drawerHead}>
+                <ListItemText primary={"Purbee"} />
+            </ListItem>
+            <List>
+                <Link to="/profile-page" style={{ textDecoration: 'none', color: "#101010" }} >
```

Comment:
this link should go to "/home"

Reply:
fixed with the commit cd4318eaab16fb2afd5151d974c60bb75e9dae8f.